azure-pipelines.yml

```diff
@@ -90,7 +90,7 @@ stages:
         displayName: "Build NuGet packages"
         inputs:
           command: pack
-          packagesToPack: 'analyzers/src/**/*.nuspec'
+          packagesToPack: 'analyzers/src/**/*.nuspec;analyzers/packaging/*.nuspec'
```

Comment:
@costin-zaharia-sonarsource  are both needed? do we have nuspecs in other folders than packaging?

Reply:
I asked the same question 😄  https://github.com/SonarSource/sonar-dotnet/pull/5143#discussion_r763871347

It's needed for CFG

Reply:
And in theory, we could move the CFG out as well